python/cuml/test/test_tsvd.py

```diff
@@ -86,3 +97,5 @@ def test_tsvd_inverse_transform(datatype, input_type):
 
     input_gdf = cutsvd.inverse_transform(Xcutsvd)
     assert array_equal(input_gdf, gdf, 0.4, with_sign=True)
+    if stream is not None:
```

Comment:
Don't we need to sync in any case? Also is the sync not needed before line 99?

Reply:
Done